flagging_site/templates/about.html

```diff
@@ -3,46 +3,72 @@
 {% block content %}
     <h2>Flagging Program</h2>
     <p>
-        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore
-        magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
-        consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
-        pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est
-        laborum.
+        Of the many services that the CRWA provides to the greater Boston community, one of those is monitoring whether it is safe to swim and/or boat in the Charles River.CRWA's
+        Flagging Program uses a system of color-coded flags to indicate whether or not the river's water quality is safe for boating at eleven boating locations between Watertown and Boston.
+        Flag colors are based on E. coli and cyanobacteria (blue-green algae) levels; blue flags indicate suitable boating conditions and red flags indicate potential health risks.
+        (<a href="https://www.crwa.org/flagging-program.html">CWRA Website</a>)
```

Comment:
I mean this sincerely: Very good use of copy+pasting! Since this is the CRWA's own website, it's not cheating or anything like that to simply use the CRWA's own language instead of getting fancy with paraphrasing. Extremely well done job with this.

Reply:
Thanks!